docs/website/content/v0.3/en/configuration/v1alpha1.md

```diff
@@ -242,18 +242,20 @@ install:
 #### files
 
 Allows the addition of user specified files.
+If an `op` value of `append` is used, the existing file will be appended.
 Note that the file contents are not required to be base64 encoded.
 
 Type: `array`
 
 Examples:
 
 ```yaml
-kubelet:
-  contents: |
-    ...
-  permissions: 0666
-  path: /tmp/file.txt
+files:
+  - contents: |
+      ...
+    permissions: 0666
+    path: /tmp/file.txt
+    op: [write,append]
```

Comment:
Originally started with this, and then found out that some error handling was needed around the value of `op`, then realized explicit is probably better than implicit.

Reply:
is this a list, or this is supposed to be single value?

Reply:
Single value ... I see the confusion ... I was trying to make the example use the "optional" notation.

Reply:
Fixed